src/app/content/reducer.ts

```diff
@@ -69,10 +68,8 @@ function reduceContent(state: State, action: AnyAction) {
         return initialState;
       }
 
-      const omitPage = omit(['page']);
-
       // book is different
-      if (!equals(omitPage(action.payload.match.params), omitPage(state.params))) {
+      if (!state.params || !isEqual(action.payload.match.params.book, state.params.book)) {
```

Comment:
you shouldn't need the first part with the `!state.params || `, if it hasn't been set then `state.params.book` will be undefined, which will never match `action.payload.match.params.book`

Reply:
But `state.params` can be null. I dont think `state.params.page` and `state.params.book` are set separately, so i kept their type as `Params | null` compared to  `state.loading` being `Partial<Params>`

Reply:
oh right ok, that makes sense